src/tests/JIT/Regression/JitBlue/Runtime_107146/Runtime_107146.cs

```diff
@@ -28,7 +28,7 @@ public class Runtime_107146
     [Fact]
     public static void TestEntryPoint()
     {
-        if (!Avx512BW.VL.IsSupported)
+        if (!Avx512BW.VL.IsSupported || !Popcnt.IsSupported)
```

Comment:
nit: Most of the other regression tests don't try to early exit; specifically because this is somewhat of an anti-pattern and trivially breaks across refactorings or misses cases like this.

Instead, we should be doing
```csharp
if (Avx512BW.VL.IsSupported)
{
    // Avx512BW usage, which also implies everything that is in the ISA hierarchy, such as popcnt
}
```